case_creation.tf

```
@@ -0,0 +1,61 @@
+module "api-case-mgmt-product" {
+  source = "git@github.com:hmcts/cnp-module-api-mgmt-product?ref=master"
+
+  api_mgmt_name = local.api_mgmt_name
+  api_mgmt_rg   = local.api_mgmt_rg
+  name = var.product_name
+  product_access_control_groups = ["developers"]
+  approval_required     = "false"
+  subscription_required = "true"
+}
+
+module "case-mgmt-api" {
+  source = "git@github.com:hmcts/cnp-module-api-mgmt-api?ref=master"
+
+  api_mgmt_name = local.api_mgmt_name
+  api_mgmt_rg   = local.api_mgmt_rg
+  revision      = "1"
+  service_url   = local.prl_api_url
+  product_id    = module.api-case-mgmt-product.product_id
+  name          = join("-", [var.product_name, "api"])
```

much easier to read:
```suggestion
  name          = "${product_name}-api"
```